praw/models/reddit/live.py

```diff
@@ -319,6 +320,32 @@ def contributor(self) -> LiveContributorRelationship:
         """
         return LiveContributorRelationship(self)
 
+    @cachedproperty
+    def stream(self) -> "LiveThreadStream":
+        """Provide an instance of :class:`.LiveThreadStream`.
+
+        Streams are used to indefinitely retrieve new updates made to a
+        live thread, like:
+
+        .. code-block:: python
+
+           for live_update in reddit.live("ta535s1hq2je").stream.updates():
+               print(live_update.body)
+
+        Updates are yielded oldest first as :class:`.LiveUpdate`. Up to 100
+        historical updates will initially be returned.
+        To only retrieve new updates starting from when the stream is
+        created, pass ``skip_existing=True``:
+
+        .. code-block:: python
+
+           live_thread = reddit.live("ta535s1hq2je")
+           for live_update in live_thread.stream.updates(skip_existing=True):
+               print(live_update.author)
```

Comment:
Could you make this indented 4 spaces instead of 3? I plan on standardizing this at some point. 
```suggestion
            live_thread = reddit.live("ta535s1hq2je")
            for live_update in live_thread.stream.updates(skip_existing=True):
                print(live_update.author)
```